Vdm 61

deerwalk opened the pull request.

prabeam committed:
VDM-61 Modified code to handle missing modules while running application
using virtual environment.

prabeam committed:
VDM-62 Made changes on the default deployment file for 'Upload Configuration'
test.

prabeam committed:
VDM-63 Modified code to fix the error in console while running Python server
side test.

prabeam committed:
Merge branch 'master' into VDM-61

manjujames commented on lib/python/vdm/server/venv_starter.py:
```diff
@@ -209,8 +209,15 @@ def start_virtual_environment(arr, packages=None, verbose=False):
             _build_virtual_environment(venv_dir, version, packages)
+        else:
+            if packages:
+                for package in packages:
```
Do you need this for loop here? packages variable will now contain the path to requirements.txt and  you can call it directly without the for loop, correct?

iarlina committed:
VDM-61: Automated Test correction in ClusterSettingsTest

iarlina committed:
Merge branch 'VDM-61' of http://github.com/deerwalk/voltdb into VDM-61

prabeam committed:
VDM-61 Added server side test to verify the status
API(/api/1.0/databases/{id}/status).

prabeam committed:
Merge branch 'VDM-61' of https://github.com/deerwalk/voltdb into VDM-61

sujmanshak committed:
VDM-64: Link to VMC will break after specifying Http Listener fixed

sujmanshak committed:
Merge branch 'VDM-61' of http://github.com/deerwalk/voltdb into VDM-61

prabeam committed:
VDM-61 Removed the unneeded 'for' loops.

prabeam committed:
Merge branch 'VDM-61' of https://github.com/deerwalk/voltdb into VDM-61

sujmanshak committed:
VDM-61: Issue with incorrect url when ip and port is passed in http-listener resolved

prabeam committed:
VDM-61 Added server side tests for APIs related to server start,stop and
status(/api/1.0/databases/{id}/servers/{id}/start
,/api/1.0/databases/{id}/servers/{id}/stop and
/api/1.0/databases/{id}/servers/{id}/status).

prabeam committed:
Merge branch 'VDM-61' of https://github.com/deerwalk/voltdb into VDM-61

sujmanshak committed:
VDM-61: Issue with recover cluster due to changes made in API resolved

iarlina committed:
VDM-61: Automation test fix in ClusterSettingsTest and ClusterConfigurationTest
